daphne_worker/src/durable/garbage_collector.rs

```diff
@@ -41,17 +41,16 @@ impl DurableObject for GarbageCollector {
                     | durable::BINDING_DAP_LEADER_COL_JOB_QUEUE
                     | durable::BINDING_DAP_HELPER_STATE_STORE => (),
                     s => {
-                        return Err(int_err(format!(
-                            "GarbageCollector: unrecognized binding: {}",
-                            s
-                        )))
+                        let message = format!("GarbageCollector: unrecognized binding: {}", s);
+                        error!("{}", message);
+                        return Err(int_err(message));
```

Comment:
`int_err()` is kind of a hack meant to work around the fact that we didn't have very good logging. Now that we do, I wonder if it would be preferable to have simpler errors. One caveat is that Cloudflare has had issues with DO logs disappearing in the past.